apps/condo/domains/notification/constants/constants.js

```diff
@@ -396,6 +399,15 @@ const MESSAGE_META = {
             messageBatchId: { required: false },
         },
     },
+    [VOIP_INCOMING_CALL_MESSAGE_TYPE]: {
+        dv: { required: true },
+        title: { required: false },
+        body: { required: true },
+        data: {
+            miniAppId: { required: true },
```

Comment:
please be more specific here with `B2CApp`

Reply:
You can discuss field names with mobile developers who will support this and pass what ever you need. this is just an example and can be changed later on.

Reply:
I know our team :)

Once it reached master, someone will use it, and then we'll not change it because of backward incompatibility 🙃

Reply:
fixed